source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -445,6 +449,31 @@ export default class NetworkStatusStore extends Store {
     } catch (error) {} // eslint-disable-line
   };
 
+  // @DECENTRALIZATION TODO: Remove
+  @observable tempCurrentEpoch: number = 0;
+  @observable tempNextEpochStart: ?string = null;
+  @action tempSetFakeData = (
+    tempCurrentEpoch: ?number,
+    tempNextEpochStart: ?string
+  ) => {
+    this.tempResetFakeData();
+    if (tempCurrentEpoch) {
+      this.tempCurrentEpoch = tempCurrentEpoch;
+    }
+    if (tempNextEpochStart) {
+      this.tempNextEpochStart = tempNextEpochStart;
+    }
+  };
+  @action tempResetFakeData = async () => {
+    this.tempCurrentEpoch = 0;
+    this.tempNextEpochStart = null;
+  };
```

Comment:
@daniloprates This is probably connected with your TODO?

Reply:
@tomislavhoracek this is being used to test and will be removed when is all set

Reply:
@daniloprates oke. I saw temp and fake vars but just to check if you still need this. 👍